apis/nucleus/src/components/listbox/ListBoxInline.jsx

```diff
@@ -209,10 +192,14 @@ export default function ListBoxInline({ options = {} }) {
           <Grid item>
             {isLocked ? (
               <IconButton tabIndex={-1} onClick={unlock} disabled={!isLocked} size="large">
-                <Lock title={translator.get('Listbox.Unlock')} />
+                <Lock title={translator.get('Listbox.Unlock')} style={iconStyle} />
```

Comment:
Does it work to use: `sx={{ fontSize: '12px' }}` instead of `style`? If it does, use that instead.